src/Microsoft.AspNetCore.Hosting/WebHostBuilder.cs

```
@@ -195,7 +197,7 @@ private IServiceCollection BuildCommonServices(out AggregateException hostingSta
                 var exceptions = new List<Exception>();
 
                 // Execute the hosting startup assemblies
-                foreach (var assemblyName in _options.HostingStartupAssemblies)
+                foreach (var assemblyName in _options.HostingStartupAssemblies.Distinct(StringComparer.OrdinalIgnoreCase))
```

So are we just going to silently ignore duplicate assemblies?